.changeset/bright-snakes-smell.md

```diff
@@ -0,0 +1,6 @@
+---
+"type-css-modules": major
+"ember-codemod-remove-ember-css-modules": minor
```

Comment:
For `ember-codemod-remove-ember-css-modules`, I indicated a minor release as I want to do a major release alongside `embroider-css-modules`.